### DISCUSSION TITLE
Docker Tags

### DISCUSSION THREAD

**Discussion by AdiAbuAli** [mrlt8/docker-wyze-bridge#172] [Q&A]
In docker for this project, what is the difference between below tags:
Latest
Main
Nightly
Edge

Which one is the latest update one? 
Which one recommended?
Which one more stable?

**Comment by mrlt8** [comment]
**Latest** is recommended as it will pull the latest [tagged release](https://github.com/mrlt8/docker-wyze-bridge/releases).

**Edge** is basically anything that gets pushed between tagged releases - documentation updates, minor patches, etc.

**Nightly** is a scheduled build which should include any dependency (ffmpeg, rtsp-simple-server, etc) updates that get updated between releases.

Main is an extra tag that is essentially the same as edge.

tl;dr - use latest unless you need an updated version of a dependency.

**Comment by AdiAbuAli** [comment]
Thanks for the clarrification...
Will pickup nightly,  if the tag looks to be last pushed as Latest or edge (e.g. edge and nightly Last pushed 15 hours ago, so will go for nightly)

Thanks,
________________________________

**Reply by mrlt8** [reply]
Again, unless you need some weird dependency update, stick with **latest**.

**Comment by AdiAbuAli** [comment]
Noted.
Thank you

Thanks,
________________________________
